.ruby-version

```diff
@@ -1 +1 @@
-2.1.2
+2.2.2
```

Comment:
Could you refrain from bumping the `.ruby-version`? That change feels orthogonal to this PR and is in fact not required.